onnx/onnx-ml.proto3

```diff
@@ -384,6 +384,7 @@ message TensorShapeProto {
       int64 dim_value = 1;
       string dim_param = 2;   // namespace Shape
     };
+    string standard_denotation = 3;
```

Comment:
did you manually add it to this file. These files are usually autogenerated (part of build process that you need to check in or just run `python onnx/gen_proto.py`, you might need to add ONNX_ML=1). Otherwise I don't get where the comment disappeared

Reply:
Simply running onnx/gen_proto.py did not update onnx-ml.proto3. I used the -m cmd line arg.